OWNERS_ALIASES

```diff
@@ -173,4 +173,10 @@ aliases:
     - eparis
     - carolynvs
     - bradamant3
+  code-of-conduct-committee:
```

Comment:
I think this is wrong

Reply:
Thanks for catching the copy pasta. Fixed